code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -60,10 +60,13 @@
 	former_jellyperson.RemoveElement(/datum/element/soft_landing)
 	
 	if(ishuman(former_jellyperson))
-		update_mail_goodies(former_jellyperson)
+		new_species.update_mail_goodies(former_jellyperson)
 	
 	return ..()
 
+/datum/species/jelly/update_mail_goodies(mob/living/carbon/human/recipient, list/mail_goodies = list(/obj/item/reagent_containers/blood/toxin))
+	return ..()
```

Comment:
There's no point doing this if you aren't passing in any new logic before you call parent, you only need to override this proc when you actually want to do stuff before/after invoking parent- this is completely redundant.

Just remove the subtype proc overall imo